main.py

```diff
@@ -1,7 +1,44 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
```

Comment:
Add blank line between different import groups

Reply:
not resolved

Reply:
There is no need to add two blank lines, add one blank line between different groups of imports to follow PEP-8 in the future